### PR TITLE
[WIP] Add Trilu op in ONNX domain from contri_ops 

### PR DESCRIPTION
**Pull request by jcwchen**
**Description**:
(Updated)
- ~Move Trilu from contri_ops to~ Add Trilu in ONNX domain; Keep original contri_ops to support backward compatibility
- Bump ONNX commit to consume Trilu support

**Motivation and Context**
https://github.com/onnx/onnx/pull/3291 ONNX has officially supported Trilu since opset_version=14

### CODE REVIEW

**Comment by thiagocrepaldi** [comment]
@BowenBao @spandantiwari FYI

**Comment by snnn** [comment]
You may keep the old one there, if it could have been used by any customer.

**Comment by faxu** [comment]
Yes, please do not remove contrib ops that have shipped previously for backwards compat. It can be registered to both official and contrib domains.

**Comment by jcwchen** [comment]
@snnn @faxu Thank you for the reminder. I have undone the removal from contri_ops. BTW, I will halt this PR now since ORT needs other changes to consume other commits from latest ONNX. It will be done in another PR.

File: onnxruntime/test/testdata/required_ops.config [hunk 1]
```diff
@@ -5,6 +5,7 @@ ai.onnx;9;Abs,Add,BatchNormalization,Cast,Clip,Concat,Constant,ConstantOfShape,C
...
 ai.onnx;11;Abs,Add,ArgMax,BatchNormalization,Cast,Clip,Concat,Constant,ConstantOfShape,Conv,Div,Exp,Expand,Flatten,Gather,Gemm,Identity,If,LayerNormalization,Log,Loop,MatMul,MatMulInteger,Max,Min,Mul,Neg,Pow,RandomUniform,Range,ReduceMean,ReduceSum,ReduceSumSquare,Relu,Reshape,Scan,SequenceConstruct,SequenceInsert,SequenceLength,Shape,Sigmoid,Slice,Softmax,Split,Sqrt,Squeeze,Sub,Sum,Tanh,Transpose,Unsqueeze,Where
 ai.onnx;12;Add,And,Cast,Concat,Constant,ConstantOfShape,Conv,CumSum,Div,Dropout,DynamicQuantizeLinear,Equal,Erf,Expand,Flatten,Gather,GatherND,Gemm,GlobalAveragePool,Identity,If,LayerNormalization,Less,Loop,MatMul,MatMulInteger,Min,Mul,NonZero,Not,Pad,Pow,RandomNormalLike,Range,ReduceMean,ReduceSum,Relu,Reshape,Shape,Slice,Softmax,SoftmaxCrossEntropyLoss,SparseSoftmaxCrossEntropy,Split,Sqrt,Squeeze,Sub,Tanh,Transpose,Unsqueeze,Where
+ai.onnx;14;Trilu
 ai.onnx;314159;Add
 ai.onnx.ml;1;ArrayFeatureExtractor,DictVectorizer,LabelEncoder,LinearClassifier,Normalizer,OneHotEncoder,TreeEnsembleRegressor,ZipMap
-com.microsoft;1;BiasGelu,Fake_FunctionOp,FastGelu,FusedGemm,FusedMatMul,Gelu,Trilu
+com.microsoft;1;BiasGelu,Fake_FunctionOp,FastGelu,FusedGemm,FusedMatMul,Gelu
```
Comment (jcwchen) [inline onnxruntime/test/testdata/required_ops.config:11]: Honestly I don't know which one (contri_ops or ONNX domain) is better for this usage.

**Comment by jcwchen** [comment]
This has been done by https://github.com/microsoft/onnxruntime/pull/7102. Close it now.

**Pull request closed without merging**